src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java

```diff
@@ -1,4 +1,4 @@
-package com.microfocus.application.automation.tools.model;
+package com.microfocus.application.automation.tools.octane.actions;
```

Comment:
If this class was serialized and introduced in previous versions, this change might break their configurations.

Reply:
This class is new for this version.

Reply:
... and it is NOT serialized.